data-prepper-plugins/otel-metrics-source/src/main/java/org/opensearch/dataprepper/plugins/source/otelmetrics/OTelMetricsGrpcService.java

```diff
@@ -40,7 +46,7 @@ public class OTelMetricsGrpcService extends MetricsServiceGrpc.MetricsServiceImp
 
 
     public OTelMetricsGrpcService(int bufferWriteTimeoutInMillis,
-                                  Buffer<Record<ExportMetricsServiceRequest>> buffer,
```

Comment:
Please be sure to verify that the original source that uses `ExportMetricsServiceRequest` is not affected by this change

Reply:
Original source is not there any more. We are replacing original source with this new behavior.